submissions/AnnaStoyano/popup/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Popup</title>
+        <link rel="stylesheet" href="./style.css">
+    </head>
+    <body>
+        <header>
+            <ul>
+               <li><a href="">Gmail</a></li>
+                <li><a href="">Images</a></li>
+                <li>
+                    <label for="main_menu" class="icon menu"></label>
+                    <input type="checkbox" id="main_menu"> 
+                    <div class="rectangle">
+                        <div class="submenu">
+                            <div tabindex="0" class="referens">
+                                <div class="icon"></div>
+                                <p>Docs</p>
+                            </div>
+                            <div tabindex="0" class="referens">
+                                <div class="icon"></div>
+                                <p>Translate</p>
+                            </div>
+                            <div  tabindex="0" class="referens">
+                                <div class="icon"></div>
+                                <p>Google+</p>
+                            </div>
+                            <div tabindex="0" class="referens">
+                                <div class="icon"></div>
+                                <p>Search</p>
+                            </div>
+                            <div tabindex="0" class="referens">
+                                <div class="icon"></div>
+                                <p>TouTube</p>
+                            </div>
+                            <div tabindex="0" class="referens">
+                                <div class="icon"></div>
+                                <p>Play</p>
+                            </div>
+                            <div tabindex="0" class="referens">
+                                <div class="icon"></div>
+                                <p>Gmail</p>
+                            </div>
+                            <div tabindex="0" class="referens">
+                                <div class="icon"></div>
+                                <p>Drive</p>
+                            </div>
+                            <div tabindex="0" class="referens">
+                                    <div class="icon"></div>
+                                    <p>Calendar</p>
+                            </div>
+                            <div tabindex="0" class="referens">
+                                <div class="icon"></div>
+                                <p>Tables</p>
+                            </div>
+                        </div>
+                        <label for="more_m" class="more">More</label>
+                        <input type="checkbox" id="more_m">    

```

Comment:
Make your popup scrollable when submenu is open.
Also, hide button More when submenu is open.